### DISCUSSION TITLE
Global sounds (e.g. music) cannot be overridden, only added to

### DISCUSSION THREAD

**Discussion by Jade-Harleyy** [FakeFishGames/Barotrauma#13608] [Ideas and suggestions]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

**What happened:** The example mod added songs to some sort of list of playable songs for each music type.

**What I expected to happen:** The example mod should've replaced the songs in the playable music types with only the songs in the mod.

**Example mod:** [ULTRAKILL OST.zip](https://github.com/Regalis11/Barotrauma/files/9997881/ULTRAKILL.OST.zip)
(Replaced music types are 'menu', 'editor', 'start', 'default', 'monster', and 'ending'.)

### Reproduction steps

1. Install and enable the mod in settings.
2. Restart the game.
3. Switch between the main menu and the editor.
4. Repeat steps 2 and 3 a couple of times.
5. Notice how the music is not always replaced with the modded versions.

### Bug prevalence

Happens every time I play

### Version

0.19.14.0

### Which operating system did you encounter this bug on?

Windows

**Comment by Rehexr** [comment]
Ive noticed this with other music mods being incompatible when i tried to enable them, so definitely an issue overall, not just with this example mod

**Comment by themanyfaceddemon** [comment]
I would also like to point out that the overwrite, even if it happens, is only for the host and not for all clients
